cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,12 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('input[type="email"]').click();
+    cy.get('input[type="email"]').type('mlemeshev@i.ua');
+    cy.get('input[type="password"]').click();
+    cy.get('input[type="password"]').type('User1234');
+    cy.get('.btn').click();
+    cy.get('#__next > nav > div > ul > li:nth-child(4) > a')
+      .should('contain', 'mlemeshev');
```

Comment:
The selector used here is very specific and could easily break if the structure of the page changes. It's recommended to use more stable selectors. For instance, you could add a data-testid attribute to the element you're trying to select.